drivers/can/can_mcan.c

```diff
@@ -822,11 +825,11 @@ int can_mcan_send(const struct device *dev, const struct can_frame *frame, k_tim
 #endif /* !CONFIG_CAN_FD_MODE */
 		.efc = 1U,
 	};
-	uint32_t put_idx;
+	uint32_t put_idx = -1;
```

Comment:
Strictly speaking, this variable is no longer holding the "put index" (at least not in the meaning from the M_CAN manual). Should we rename it?

Reply:
Let's keep the name for now.